src/components/Popover/Popover.module.scss

```
@@ -68,19 +68,19 @@
 
     // Arrows
     .isRootAtTop > .arrow {
-        top: 100%;
+        top: calc(100% - 1px);
```

Creating a variable (or a constant) makes the code easier to understand:

```suggestion
        top: calc(100% - #{$arrow-safe-overlap});
```

And in the `_theme.scss`:

```scss
$arrow-safe-overlap: 1px;
```

---

Maybe the name could be `$arrow-safe-rendering-overlap` so it's even clearer. Something like that.